src/components/commons/inputs/textAreas/TextArea.tsx

```diff
@@ -0,0 +1,30 @@
+import { ChangeEvent, TextareaHTMLAttributes, useState } from "react";
+import * as S from "./TextArea.style";
+
+export interface TextAreaPropsTypes extends TextareaHTMLAttributes<HTMLTextAreaElement> {
+  maxLength?: number;
+  placeholder: string;
+}
+
+const TextArea = ({ maxLength, placeholder }: TextAreaPropsTypes) => {
+  const [inputValue, setInputValue] = useState("");
+  const [inputCount, setInputCount] = useState(0);
+  const handleOnInput = (e: ChangeEvent<HTMLTextAreaElement>) => {
+    const value = e.target.value;
+    setInputValue(value);
+    setInputCount(value.length);
+  };
+  return (
+    <S.TextAreaWrapper>
+      <S.TextAreaInput
+        value={inputValue}
+        onChange={handleOnInput}
+        maxLength={maxLength}
+        placeholder={placeholder}
+      ></S.TextAreaInput>
```

Comment:
p4) 여기 안에 들어가는 content가 없어서 self closing 태그로 바꾸면 좋을 것 같아요! Input도 마찬가지로!
```suggestion
    <S.TextAreaWrapper>
      <S.TextAreaInput
        value={inputValue}
        onChange={handleOnInput}
        maxLength={maxLength}
        placeholder={placeholder}
       />
```